examples/models/llava_encoder/install_requirements.sh

```diff
@@ -19,4 +21,7 @@ pip install bitsandbytes -I
 # For example, torch version required from llava is older than ExecuTorch.
 # To make both work, recover ExecuTorch's original dependencies by rerunning
 # the install_requirements.sh.
-./install_requirements.sh
+pip install --force-reinstall torch==2.3.0 --index-url https://download.pytorch.org/whl/test/cpu
```

Comment:
Per my investigation in [T184361451](https://www.internalfb.com/intern/tasks/?t=184361451), I think we should avoid using `-I` as in `pip install bitsandbytes -I`